src/dotnet/commands/dotnet-add/dotnet-add-proj/Program.cs

```diff
@@ -62,7 +62,7 @@ public override int Run(string fileOrDirectory)
 
         private void AddProject(SlnFile slnFile, string projectPath)
         {
-            var projectPathNormalized = PathUtility.GetPathWithBackSlashes(projectPath);
+            var projectPathNormalized = PathUtility.GetPathWithDirectorySeparator(projectPath);
 
             if (slnFile.Projects.Any((p) =>
                     string.Equals(p.FilePath, projectPathNormalized, StringComparison.OrdinalIgnoreCase)))
```

Comment:
is this working on linux? when you are adding you are converting to backslashes and now you compare with something which might technically have regular slashes